plenum/test/node_catchup/helper.py

```diff
@@ -37,11 +37,21 @@ def chk_ledger_and_state(first_node, second_node, ledger_id):
         else:
             logger.debug("Excluding check_last_ordered_3pc check")
 
+        if exclude_from_check and 'check_view_no' not in exclude_from_check:
+            check_view_no(node, n)
+        else:
+            logger.debug("Excluding check_view_no check")
+
         if exclude_from_check and 'check_seqno_db' not in exclude_from_check:
             check_seqno_db_equality(node.seqNoDB, n.seqNoDB)
         else:
             logger.debug("Excluding check_seqno_db_equality check")
 
```

Comment:
Can we check last_ordered on all instances, not backup only?

Reply:
fixed